nupic/research/frameworks/pytorch/imagenet/imagenet_tune.py

```diff
@@ -195,13 +192,13 @@ def _create_workers(self, config):
 
         self._process_config(config)
 
-        experiment_class = config["experiment_class"]
+        self.experiment_class = config["experiment_class"]
```

Comment:
I would initialize this field at the beginning of `_setup` method instead `_create_workers` method

Reply:
Good point.